fdbserver/workloads/IncrementalBackup.actor.cpp

```diff
@@ -51,19 +54,50 @@ struct IncrementalBackupWorkload : TestWorkload {
 		return _start(cx, this);
 	}
 
-	virtual Future<bool> check(Database const& cx) { return true; }
+	virtual Future<bool> check(Database const& cx) {
+		if (clientId || !waitVersion) {
+			return true;
+		}
+		return _check(cx, this);
+	}
+
+	ACTOR static Future<bool> _check(Database cx, IncrementalBackupWorkload* self) {
+		state Reference<IBackupContainer> backupContainer;
+		state UID backupUID;
+		int waitResult =
+		    wait(self->backupAgent.waitBackup(cx, self->tag.toString(), false, &backupContainer, &backupUID));
+		TraceEvent("IBackupCheckWaitResult").detail("Result", waitResult);
```

Comment:
How is this `waitResult` used? Why not just skip the call to `waitBackup` and enter the loop which calls `describeBackup`?

Reply:
`waitResult` is just used for logging purposes, but the call to `waitBackup` is necessary in order to get the backupContainer object filled in